charts/jenkins/templates/jenkins-controller-statefulset.yaml

```diff
@@ -140,7 +140,13 @@ spec:
         - name: jenkins
           image: "{{ .Values.controller.image }}:{{ .Values.controller.tag }}"
           imagePullPolicy: "{{ .Values.controller.imagePullPolicy }}"
-          {{- if .Values.controller.httpsKeyStore.enable }}
+          {{- if .Values.controller.overrideArgs }}
+          args: [
+            {{- range $overrideArg := .Values.controller.overrideArgs }}
+              "{{- $overrideArg }}",
```

Comment:
could you add a unit test for this please? also needs a changelog entry and the chart version bumped

Reply:
@timja is it ok now?